langchain/src/retrievers/multi_query.ts

```diff
@@ -0,0 +1,123 @@
+import { LLMChain } from "../chains/llm_chain.js";
+import { PromptTemplate } from "../prompts/prompt.js";
+import { Document } from "../document.js";
+import { BaseLLM } from "../llms/base.js";
+import { BaseOutputParser } from "../schema/output_parser.js";
+import { BaseRetriever } from "../schema/retriever.js";
+import { CallbackManagerForRetrieverRun } from "../callbacks/index.js";
+
+interface LineList {
+    lines: string[];
+}
+
+class LineListOutputParser extends BaseOutputParser<LineList> {
+    lc_namespace: string[] = ["langchain", "output_parsers", "linelist"];
+    async parse(text: string): Promise<LineList> {
+        const lines = text.trim().split("\n");
+        return { lines };
+    }
+    getFormatInstructions() {
```

Comment:
This can just return `""`, there's no specific formatting instructions for a prompt used here that are from this output parser.

Reply:
Thanks, I didn't know what these instructions were intended for in the first place!